src/query/functions/src/scalars/variant.rs

```diff
@@ -126,131 +124,6 @@ pub fn register(registry: &mut FunctionRegistry) {
         }),
     );
 
-    registry.register_passthrough_nullable_1_arg::<BooleanType, VariantType, _, _>(
```

Comment:
why do we remove these functions?

Reply:
Because it will be tedious to implement parse_json for decimals, and also because having only primitive numbers overload is inconsistent.